testdata/baselines/reference/submoduleAccepted/compiler/commonJsExportTypeDeclarationError.errors.txt.diff

```diff
@@ -21,21 +18,17 @@
      }
      
 -==== ./types1.ts (2 errors) ====
-+==== types1.ts (2 errors) ====
++==== types1.ts (1 errors) ====
      import test from "./test";
-+                     ~~~~~~~~
-+!!! error TS2306: File 'test.js' is not a module.
      export type test
 -                    
```

Comment:
Do we have a parser bug?

Reply:
I suspect it's a difference in error reporting ("Type expected" and "'=' expected" appear to be on the same location), but I'll confirm.

Reply:
Yep. The code is working as expected in Corsa but not in Strada. The two messages are both on the EOF token in Corsa. I'm not sure why they evade position-based deduping in Strada.